test/uk/gov/hmrc/traderservices/support/UnitSpec.scala

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Looks like this file is a duplicate of UnitSpec under it/uk/gov/hmrc/traderservices/support/UnitSpec.scala . Would it make more sense to create a commons package and move it there so that if a change is needed we don't have to update in 2 places? If the files are diff then ignore this comment.

Reply:
This duplication is intentional